middleware/jwt.go

```diff
@@ -42,7 +42,7 @@ func init() {
 	}
 }
 
-// From https://github.com/dgrijalva/jwt-go/pull/139 it is understood
+// From https://github.com/form3tech-oss/jwt-go/pull/139 it is understood
```

Comment:
This change does not make sense. The PR is only available in the dgrijalva repo. We can keep the comment as-is or modify the comment to not refer to any PR altogether.